examples/features/driven_dynamics/efield-water/README.md

```diff
@@ -0,0 +1,157 @@
+
+<ins>To summarize</ins>: any physical quantity (energy, forces, stresses, dipole, etc ...) only depend on the nuclear coordinates $\mathbf{R}$, but the equations of motions (for $\mathbf{R}$) are modified by the presence of the electric field.
+
+
+## Overview 
+#### Electric field
+The electric field-driving will be performed on top of a ```NVE``` simulation.
+In this case the adopted electric field is a **plane wave** with a **gaussian envelope function**.
+The **frequency** $\omega$ and **direction** $\hat{\mathbf{E}}_{amp}$ of the plane wave (```115THz``` along the ```y```-axis) are chose to be resonant with one of the three vibrational mode of the system.
+The **amplitude** $|\mathbf{E}_{amp}|$ of the field is chose to be not too intense.
+The gaussian envelope function is determined by its **mean** $\mu$ (when the electric field will reach its maximum value) and **std** $\sigma$ (the duration of the pulse).
+This is the analytic expression of the applied electric field is:
+$$
+\mathbf{E}\left(t\right) = \mathbf{E}_{amp}\cos \left( \omega t + \phi \right) \frac{1}{{\sigma \sqrt{2\pi}}} e^{-\frac{{(x - \mu)^2}}{{2\sigma^2}}}
+$$
+
+The parameters that define the shape of the electric field are defined in the ```ensemble``` field in the ```input.xml``` file. 
+Some comments are provided in that file as well.
+
+The parameters of the electric field are provided in the `input.xml` as follows (their names are self-explaining):
+```xml
+<motion mode='dynamics'> 
+    <dynamics mode='eda-nve'>                  <!-- Electric Dipole Approximation (EDA) on top of a NVE simulation -->
+        <timestep units='femtosecond'> 1 </timestep>
+        <efield>
+            <amp> [ 0 , 5e-2 , 0 ] </amp>             <!-- amplitude of the electric field pulse in a.u. -->
+            <freq units="thz"> 115 </freq>            <!-- frequency of the electric field pulse in THz -->
+            <peak units="picosecond"> 0.2 </peak>     <!-- peak of the electric field pulse -->
+            <sigma units="picosecond"> 0.05 </sigma>  <!-- width (standard deviation of the gaussian) of the electric field pulse -->
+        </efield>
+        <bec mode="file"> bec.txt  </bec>
+    </dynamics>
+</motion>
+```
+
+#### Dipole and Born Effective Charge tensors
+The dipole $\mathbf{d}$ of the system is not actually computed in this example.
+In the same way we do not *actually* need the energy to integrate Hamilton's equations of motion but only the forces, we just need the Born Effective Charge tensors $Z^*_{ij}$ to run a light-driven simulation, i.e. the derivative of the dipole $\mathbf{d}$ w.r.t. nuclear coordinates $\mathbf{R}$:
+$$
+Z^*_{ij} = \frac{1}{q_e} \frac{\partial d_i}{\partial R_j}
+$$
+
+In this example we will provide the Born Effective Charge tensors $Z^*_{ij}$ using the input file ```bec.txt```, assuming (for semplicity) that their value is constant for each nuclear configuration $\mathbf{R}$.
```

Comment:
I think it'd be much better to provide these in the input file rather than through an external file. 
If for whatever reason you want to stick to an external file, explain they can call it as they like and the name is specified in the <bec> field.

Reply:
I would provide the BEC tensors using an external file because they could be really large for bigger systems.
In this `README.md` I have explained how to provide these tensors using an external file and directly into the `input.xml` file.

Reply:
I agree with Elia.